docs/reference/platforms/macOS/xcode.rst

```diff
@@ -21,11 +21,15 @@ By default, apps will be both signed and notarized when they are packaged.
 Packaging format
 ================
 
-Briefcase supports two packaging formats for a macOS Xcode project:
+Briefcase supports three packaging formats for a macOS Xcode project:
```

Comment:
This text should always be identical to the same section in app.rst. Could it be shared using an `.. include` directive?

Reply:
Unfortunately, it's *just* different enough to make this impossible - the command to invoke packaging is different.

Reply:
In that case, the same updates should be applied to this file.